cmd/network/verification.go

```diff
@@ -790,3 +801,60 @@ func printVersion() {
 	}
 	log.Println(fmt.Sprintf("Using osd-network-verifier version %v", version))
 }
+
+// Put Cluster to LS if it can't access pd (OR) DMS
+func putLimitedSupport(ClusterId string) error {
+
+	connection, err := utils.CreateConnection()
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err = connection.Close(); err != nil {
+			fmt.Printf("Cannot close the connection: %q\n", err)
+			os.Exit(1)
+		}
+	}()
+
+	summary := "Your cluster requires you to take action. SRE has observed that there have been changes made to the network configuration which impacts normal working of the cluster, including lack of network egress to these internet-based resources which are required for the cluster operation and support. Please revert changes, and refer to documentation regarding firewall requirements for PrivateLink clusters: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/prepare_your_environment/rosa-sts-aws-prereqs#osd-aws-privatelink-firewall-prerequisites_rosa-sts-aws-prereqs"
+	limitedSupportBuilder := cmv1.NewLimitedSupportReason().Summary("Cluster is in Limited Support due to unsupported network configuration").Details(summary).DetectionType("manual")
+	limitedSupport, err := limitedSupportBuilder.Build()
+	if err != nil {
+		return fmt.Errorf("failed to build new limited support reason: %w", err)
+	}
```

Comment:
**suggestion:** We can likely make this more descriptive and also list blocked egresses. I suggest running this through region leads / management for approval afterwards.